pol-core/pol/module/osmod.cpp

```diff
@@ -1400,6 +1460,215 @@ BObjectImp* OSExecutorModule::mf_GetEnvironmentVariable()
   }
 }
 
+BObjectImp* OSExecutorModule::mf_SendEmail()
+{
+  if ( Core::settingsManager.email_cfg.url.empty() )
+    return new BError( "Email settings are not configured" );
+
+  Core::UOExecutor& this_uoexec = uoexec();
+
+  if ( this_uoexec.pChild == nullptr )
+  {
+    const String *from, *subject, *body, *contentType;
+    BObjectImp *recipient, *bcc;
+
+    if ( getStringParam( 0, from ) && getParamImp( 1, recipient ) && getStringParam( 2, subject ) &&
+         getStringParam( 3, body ) && getParamImp( 4, bcc ) && getStringParam( 5, contentType ) )
+    {
+      if ( !this_uoexec.suspend() )
```

Comment:
Multiple error returns exist which do not revive the script. So if anything happens the script is dead

Reply:
Addressed in 1d3bee66f94da60ca09cc994abde4f253836d667